test/gas/gas-costs.json

```diff
@@ -1,4 +1,4 @@
 {
 	"Root Domain Price": "475352",
-	"Subdomain Price": "469054"
+	"Subdomain Price": "478990"
```

Comment:
are we sure this is correct? only +9k from all those changes?

Reply:
not sure this is 100% accurate, getting differing numbers